UIKitDZ/UIKitDZ/SingInViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  ViewController.swift
+//  UIKitDZ
+//
+//  Created by Maxim on 22.09.2022.
+//
+
+import UIKit
+/// Контроллер входа в приложение
+final class SingInViewController: UIViewController {
+
+    let emailTextField = UITextField()
```

Comment:
```suggestion
   private  let emailTextField = UITextField()
```
private - везде ниже